config/locales/ru.yml

```diff
@@ -0,0 +1,381 @@
+      edit: Редактировать
+      errors:
+        user_ids_length: Выберите хотя-бы одного пользователя.
+      form:
+        content_label: :thredded.private_posts.form.content_label
+        create_btn: :thredded.private_posts.form.create_btn
+        create_btn_submitting: :thredded.private_posts.form.create_btn_submitting
+        title_label: :thredded.topics.form.title_label
+        title_placeholder_new: Необходимо написать предмет переписки
+        title_placeholder_start: Начать новую переписку
+        update_btn: Обновить
+        update_btn_submitting: :thredded.private_posts.form.update_btn_submitting
+        users_label: Участники
+        users_placeholder: Выберите пользователей для участия в переписке
+      no_private_topics:
+        create_btn: Начать вашу первую приватную переписку
+        title: У вас нет личных сообщений.
+      updated_notice: Заголовок обновлен
+    recent_activity: Активность
+    search:
+      form:
+        btn_submit: :thredded.search.form.label
+        label: Поиск
+        placeholder: Поиск тем или записей
+    time_ago: "%{time} назад"
+    topics:
+      delete_confirm: Вы уверены что хотите удалить эту тему? Это безвозвратно.
+      delete_topic: Удалить тему
+      deleted_notice: Тема удалена
+      edit: Редактировать тему
+      follow: Наблюдать за темой
+      followed_by: 'Наблюдаю'
+      followed_by_noone: Никто не наблюдает за этой темой
+      followed_notice: Вы следите за темой
+      following:
+        manual: Вы следите за этой темой.
+        mentioned: Вы следите за этой темой, потому что упомянуты в ней.
+        posted: Вы следите за темой, потому что Вы писали в ней.
+      form:
+        categories_placeholder: Категории
+        content_label: :thredded.posts.form.content_label
+        create_btn: Создать новую тему
+        messageboard_label: Форум
+        title_label: Заголовок
+        title_placeholder: :thredded.topics.form.title_label
+        title_placeholder_start: Начать новую тему
+      currently_online: Онлайн
+      last_active_html: Последняя активность %{time_ago}
+      posted_in_topic_html: Сделан пост в %{topic_link}
+      posts_count:
+        one: Сделан пост
+        other: Сделано постов %{count} раз
+      recent_activity: :thredded.recent_activity
+      started_topic_html: Начато %{topic_link}
+      started_topics_count:
+        one: Начата тема
+        other: Начато %{count} тем
+      user_posted_in_topic_html: "%{user_link} сделал запись в %{topic_link}"
+      user_since_html: Пользователь был %{time_ago}
+      user_started_topic_html: "%{user_link} начал %{topic_link}"
+      unfollow: Не следить за темой
+      unfollowed_notice: Вы не следите за темой
+      mark_as_unread: Пометить как непрочитанное
+      not_following: не отслеживаю
+      locked:
+        label: Блокировать
+      sticky:
+        label: Закрепить
+    users:
+      currently_online: Онлайн
+      user_started_topic_html: Сделана запись
+  activerecord:
```

Comment:
All the keys from here and below are [part of rails-i18n](https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/ru.yml) and we should not be duplicating them here.